### PR TITLE
Side menu

### PR DESCRIPTION
**Pull request by ayebaishmo**
# in this branch
- Add javascript file o the project
- Add side menu html
- Design side menu with css
- Add functionality with javascript to close and open the menu

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -51,6 +57,99 @@ body {
...
+  margin-top: -10px;
+  margin-left: -25px;
+}
+
+#sideMenu {
+  display: none;
+  position: fixed;
+  top: 92px;
+  z-index: 3;
+  background-color: #3c3a39;
```
Comment (dasileker) [inline index.css:97]: - [ ] the menu should be visible only for the mobile version not for the desktop.

mmm

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @ayebaishmo,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by oxenprogrammer — CHANGES_REQUESTED** [review]
Hi @ayebaishmo, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has `failing linter test`. Please let us make sure that our CI is all `green` before requesting another code review.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.css [hunk 2]
```diff
@@ -51,6 +57,99 @@ body {
...
+  margin-top: 25px;
+  margin-bottom: 25px;
+}
+
+.works_det > div > ul {
+  margin-top: -10px;
+  margin-left: -25px;
+}
+
+#sideMenu {
```
Comment (dasileker) [inline index.css:92]: - [ ] The hamburger menu button is displayed on the desktop view, please fix it.

mmm

**Review by dasileker — CHANGES_REQUESTED** [review]
### **Hi @ayebaishmo,** 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.css (hunk 2 above)
Comment (ayebaishmo) [reply index.css:92]: Hello @dasileker I fixed that after you marked my request review invalid.
I think you didnot update your local file or forgot to clone it again. Here is the screen shot
hamburgscreenshot

Comment (dasileker) [reply index.css:92]: we don't check the codes if the linters are passing so after fixing linters we are supposed to download the repo and make the review, if I didn't clone how I'm supposed to see the issue, but anyway.

**Review by dasileker — APPROVED** [review]
## **Hi @ayebaishmo,** 



## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 


Approved
 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**